zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/UserTaskQueryController.java

```diff
@@ -81,4 +84,26 @@ public ResponseEntity<Object> getByKey(@PathVariable("userTaskKey") final Long u
       return RestErrorMapper.mapProblemToResponse(problemDetail);
     }
   }
+
+  @GetMapping(
+      path = "/{userTaskKey}/form",
+      produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_PROBLEM_JSON_VALUE})
+  public ResponseEntity<Object> getFormByUserTaskKey(
+      @PathVariable("userTaskKey") final Long userTaskKey) {
+    try {
+      final Long formKey = userTaskServices.getByKey(userTaskKey).formKey();
+
+      if (formKey == null) {
+        return ResponseEntity.noContent().build();
```

Comment:
🔧 Let's also add this response code to the OpenAPI spec so people are aware that this is what happens if the user task has no form.